src/schemas/json/jj.json

```diff
@@ -0,0 +1,136 @@
+{
```

Comment:
I only saw the path of this file now. When you asked where I thought we should put it, I thought you were talking about the SchemaStore repo :) In our repo, I think we should eventually generate it in `jj debug config-schema` or something. For now, I think `src/config-schema.json` sounds good. I think we should already add a ` jj debug config-schema` command that just prints the contents of `src/config-schema.json`. By "already", I don't mean that it necessarily has to go in this PR.

Reply:
Sure, moved. I did notice the original name seemed a little deep in the config structure. =)